### PR TITLE
 update describe of auto-increment

### PR DESCRIPTION
**Pull request by kissmydb**
auto-increment cannot guarantee self-increment in multiple tidb-server

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: mysql-compatibility.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ aliases: ['/docs-cn/dev/reference/mysql-compatibility/']
 
 ## 与 MySQL 有差异的特性详细说明
 
-### 自增 ID
+### [自增 ID](/auto-increment.md)
 
-- TiDB 的自增列仅保证自增且唯一、但不保证自动分配的值的连续性，建议不要将缺省值和自定义值混用，若混用可能会收 `Duplicated Error` 的错误信息。
+- TiDB 的自增列仅保证唯一，也能保证在单个 `tidb-server` 中自增，但不保证多个 `tidb-server` 中自增，不保证自动分配的值的连续性，建议不要将缺省值和自定义值混用，若混用可能会收 `Duplicated Error` 的错误信息。
```
Comment (TomShawn) [inline mysql-compatibility.md:44]:
```suggestion
- TiDB 的自增列仅保证唯一，也能保证在单个 TiDB server 中自增，但不保证多个 TiDB server 中自增，不保证自动分配的值的连续性，建议不要将缺省值和自定义值混用，若混用可能会收 `Duplicated Error` 的错误信息。
```

File: mysql-compatibility.md [hunk 2]
```diff
@@ -39,9 +39,9 @@ aliases: ['/docs-cn/dev/reference/mysql-compatibility/']
 
 ## 与 MySQL 有差异的特性详细说明
 
-### 自增 ID
+### [自增 ID](/auto-increment.md)
```
Comment (TomShawn) [inline mysql-compatibility.md:42]: @YiniXu9506 Can we add a link in title?

File: mysql-compatibility.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ aliases: ['/docs-cn/dev/reference/mysql-compatibility/']
...
 
-- TiDB 的自增列仅保证自增且唯一、但不保证自动分配的值的连续性，建议不要将缺省值和自定义值混用，若混用可能会收 `Duplicated Error` 的错误信息。
+- TiDB 的自增列仅保证唯一，也能保证在单个 `tidb-server` 中自增，但不保证多个 `tidb-server` 中自增，不保证自动分配的值的连续性，建议不要将缺省值和自定义值混用，若混用可能会收 `Duplicated Error` 的错误信息。
 
 - TiDB 在工程实现上会在每一个 tidb-server 实例上缓存一段 ID 的值用于给表的自增列分配值，缓存 ID 的个数由表的 `AUTO_ID_CACHE` 确定，默认值：30000，请特别注意：自增列和`_tidb_rowid`都会消耗缓存的 ID，如果 `INSERT` 语句中所要求的连续的 ID 个数大于 `AUTO_ID_CACHE` 的值时系统会自动调整 `AUTO_ID_CACHE` 的值以确保该语句能正常执行。
```
Comment (tangenta) [inline mysql-compatibility.md:46]: https://github.com/pingcap/docs-cn/pull/3163/files#diff-7028919502ad5a35697a652366fdafadR46
I suggest removing this paragraph.

File: mysql-compatibility.md [hunk 4]
```diff
@@ -37,9 +37,9 @@ aliases: ['/docs-cn/dev/reference/mysql-compatibility/']
 
 ## 与 MySQL 有差异的特性详细说明
 
-### 自增 ID
+### [自增 ID](/auto-increment.md)
```
Comment (TomShawn) [inline mysql-compatibility.md:40]:
```suggestion
### 自增 ID
```

**Pull request merged**